dispatcher/dispatcher.go

```diff
@@ -230,7 +230,7 @@ loop:
 		select {
 		case f := <-d.taskChan:
 			f()
-			break
+			// break //ineffective break statement in Golang. Leave it here for visual understanding.
```

Comment:
```suggestion
```
we can just remove break

Reply:
For sure. Thanks.
Do you have an idea when these errors "Develop / Develop (Go1.18,ubuntu-20.04) (pull_request)" will be fixed?